cscs-checks/prgenv/affinity_check.py

```diff
@@ -21,166 +35,520 @@ def __init__(self, variant):
         self.sourcesdir = os.path.join('src/affinity_ref')
         self.prebuild_cmds = ['git clone https://github.com/vkarak/affinity']
```

Comment:
why not using /apps/common/UES/reframe/resources/ here ? you can always the link to the repo as a comment for non CSCS users

Reply:
@jgphpc I would like that the tests are self contained, so for me it's better if it doesn't use the resources dir.